app/views/books/_book.html.erb

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <td class="px-3"><%= book.name %></td>
+  <td class="px-3"><%= book.isbn %></td>
+  <td class="px-3"><%= link_to("Edit", edit_book_path(book)) %></td>
+  <td><%= link_to "Delete", book_path(book),
+              method: :delete,
+              data: { confirm: 'Are you sure?' } %></td>
+</tr>
```

Comment:
This is how we'd do the formatting for multiple lines in a view in App:

```erb
  <td>
    <%=
      link_to("Delete", book_path(book),
        method: :delete,
        data: { confirm: "Are you sure?" }
    %>
  </td>
```

Note the double quotes around the confirm message for consistency.

On a separate note, from a UX experience I've always found it valuable to provide some context for the confirm message.

The current question is pretty open "Are you sure?" - am I sure of what? Deleting? Deleting what?

I'd opt for something more specific like `"Are you sure you want to delete '#{ book.name }'?"`

Which would give a prompt like:

> Are you sure you want to delete 'Harry Potter and the chamber of cool magic stuff'?"

Now we have an unambiguous message that allows the reader to know both the action and the target. So if you click on the wrong delete button by accident you can see the book name right there.